hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -663,6 +663,16 @@ public class HoodieWriteConfig extends HoodieConfig {
           + "will not print any configuration which contains the configured filter. For example with "
           + "a configured filter `ssl`, value for config `ssl.trustore.location` would be masked.");
 
+  public static final ConfigProperty<Boolean> ROLLBACK_BACKUP_ENABLED = ConfigProperty
+      .key("hoodie.rollback.backup.enabled")
```

Comment:
can we add "timeline" may be to the name. not very apparent. seems like entire data might be backedup.

Reply:
hoodie.rollback.instant.backup.enabled should be better name?